Source/WebKit/Platform/spi/Cocoa/BlockSPI.h

```diff
@@ -23,7 +23,8 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#if USE(APPLE_INTERNAL_SDK)
+// FIXME: COMPILER(APPLE_CLANG) is needed until <rdar://120571425> is fixed.
```

Comment:
I think this:

```
#if USE(APPLE_INTERNAL_SDK) && COMPILER(APPLE_CLANG)
```

Reads better than this (need to also add the `defined(__clang__)` check for context):

```
#if USE(APPLE_INTERNAL_SDK) && defined(__clang__) && defined(__apple_build_version__)
```

On the other hand, defining `WTF_COMPILER_APPLE_CLANG` in `Compiler.h` is self-documenting because of where it happens:

```
#if defined(__clang__)
[...]
#if defined(__apple_build_version__)
#define WTF_COMPILER_APPLE_CLANG 1
#endif
[...]
#endif
```

Reply:
OK, no worries